Cargo.toml

```diff
@@ -5,6 +5,24 @@ authors = ["Nathaniel McCallum <npmccallum@redhat.com>"]
 license = "Apache-2.0"
 edition = "2018"
 
+homepage = "https://github.com/enarx/iocuddle"
+repository = "https://github.com/enarx/iocuddle"
+description="Library for building runtime-safe linux `ioctl()` interfaces"
+readme = "README.md"
+
+keywords = ["linux", "ioctl"]
+categories = ["os", "api-bindings"]
+
+exclude = [ ".gitignore", ".github/*" ]
+
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
+[badges]
+# See https://doc.rust-lang.org/cargo/reference/manifest.html#the-badges-section
+github = { repository = "enarx/iocuddle", workflow = "test" }
+#github = { repository = "enarx/iocuddle", workflow = "lint" }
```

Comment:
This is very unfortunate. Is there a workaround? Could we fix cargo to support multiple workflows?

Reply:
Okay. It looks like fixing this would be hard. Let's remove the lint line altogether.